Sources/Errors/AlertCoordinator.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  AlertCoordinator.swift
+//  FuntastyKit
+//
+//  Created by Milan Strnad on 03/03/17.
+//  Copyright © 2017 The Funtasty s.r.o. All rights reserved.
+//
+
+import UIKit
+
+public final class AlertCoordinator {
+
+    let parentViewController: UIViewController
+    weak var viewController: UIAlertController?
+
+    var error: Error?
+
+    var title: String?
+    var message: String?
+    var actions: [UIAlertAction]?
+
+    // MARK: - Inits
+
+    public init(parent: UIViewController, error: Error) {
+        self.parentViewController = parent
+        self.error = error
+    }
+
+    public init(parent: UIViewController, title: String?, message: String?, actions: [UIAlertAction]? = [UIAlertAction(title: NSLocalizedString("OK", comment: "OK"), style: .default)]) {
+        self.parentViewController = parent
+        self.title = title
+        self.message = message
+        self.actions = actions
+    }
+}
+
+public extension DefaultCoordinator {
+    public func navigateToAlert(for error: Error) {
+        guard let viewController = self.viewController else {
+            return
+        }
+        let alertCoordinator = AlertCoordinator(parent: viewController, error: error)
+        alertCoordinator.start()
+    }
+
+    public func navigateToAlert(title: String, message: String) {
+        guard let viewController = self.viewController else {
+            return
+        }
+        let alertCoordinator = AlertCoordinator(parent: viewController, title: title, message: message)
+        alertCoordinator.start()
+    }
+
+    public func navigateToAlert(title: String, message: String, actions: [UIAlertAction]?) {
```

Comment:
There is also newly introduced `ErrorAction` struct not requiring UIKit, shouldn't it be used here too?